src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/SnackBar.android.cs

```diff
@@ -24,7 +25,7 @@ internal void Show(Page sender, SnackBarOptions arguments)
 				snackBarView.SetBackgroundColor(arguments.BackgroundColor.ToAndroid());
```

Comment:
(line 20) `Platform.GetRenderer(sender)` can return null. #959
Since `GetRenderer` is Xamarin.Forms method we should assume that is can return null. 

So, should this possible `NRE` be handled in this PR as well? At least like this:
```cs
var view = Platform.GetRenderer(sender)?.View ?? throw new NullReferenceException("renderer cannot be null");
```